### ISSUE TITLE
Fix falling unit tests

### ISSUE THREAD

**Issue by PetraKinder** [fri-team/Swapify#238]

**Comment by marektavac** [comment]
What is the difference with task #236 , may be this is a duplicate?

**Comment by BranislavTvrdy** [comment]
Yes, I believe this one to be a duplicate. I created task #236 and then @PetraKinder made another task. I told her on a meeting there is one already.
